Meadowlark-js/packages/meadowlark-core/src/metaed/MetaEdValidation.ts

```diff
@@ -77,16 +77,57 @@ export function validateQueryParametersAgainstSchema(
 ): string[] {
   // TODO: RND-67 removed the Joi-specific code to report invalid query parameters
   // RND-307 will restore as a part of understanding Ajv error metadata
+
+  let errors: string[] = [];
+
   const schema = {
     ...metaEdModel.data.meadowlark.jsonSchema,
     // Need to relax the validation such that no fields are "required"
     required: [],
   };
 
+  const parameterKeys = Object.keys(queryParameters);
+
+  /**
+   * Number and boolean are come through as strings, which causes ajv to fail validation of parameters
+   * This loops through all query parameters and checks the type against the schema, if the type is numeric or boolean
+   * it will attempt convert the value. If the conversion is good, we set the value back and ajv will validate the schema
+   * If the value can't be converted (i.e. a word is provided for a numeric value, we leave it and let ajv invalidate
```

Comment:
Are their cases where this could be a false-negative? Specifically where a number is provided in the query but is meant as a string for a string field?

Reply:
This worked just fine: 
```
### 
POST http://localhost:3000/local/v3.3b/ed-fi/students
content-type: application/json
authorization: bearer {{authToken4}}

{ 
    "studentUniqueId": "1",
    "firstName": "Hello",
    "lastSurname": "World",
    "birthDate": "2001-01-01",
    "birthCountryDescriptor": "uri://ed-fi.org/CountryDescriptor#AD3"
}

###
GET http://localhost:3000/local/v3.3b/ed-fi/students?studentUniqueId=1
authorization: bearer {{authToken4}}
```